guides/05_Simulcast.md

```diff
@@ -18,7 +18,15 @@ Simulcast support is built into the WebRTC library in the majority of browsers.
 
 The recommended resolution in JS SDK is 1280x720. This resolution will provide the most flexibility and allow subscribers to maintain smoother video quality transitions.  In certain circumstances, like mobile browsers, the input resolution may not be high enough to support all desired simulcast streams, so the logic will adapt appropriately to send as many as possible.
 
-The SimulcastUplinkPolicy configures RTCRtpSender to have three encoding layers, but only ever enables two of them.  Which two are enabled is based on a variety of factors.   Experiments show that configuring RTCRtpSender with three encoding layers up front and dynamically enabling and disabling layers provides a better experience and reduces the burden of having to dynamically  manage capture resolution and encoding parameters.
+The SimulcastUplinkPolicy configures RTCRtpSender to have three encoding layers, but only ever enables two of them.  Which two are enabled is based on a variety of factors. Experiments show that configuring RTCRtpSender with three encoding layers up front and dynamically enabling and disabling layers provides a better experience and reduces the burden of having to dynamically  manage capture resolution and encoding parameters.
+
+### Browser compatibility
+
+|Browser              |Supported versions  |
+|---                  |---                 |
+|Chrome               |76+                 |
```

Comment:
If we want to add a support table, we want a full coverage of what Chromium browser we support (Chrome, Android Chrome, Edge, etc). :)